build-system/tasks/e2e/describes-e2e.js

```diff
@@ -112,6 +114,9 @@ function describeEnv(factory) {
             }
           }
         });
+        totalPromise = totalPromise.then(() => {
```

Comment:
Could just return `totalPromise.then`. Can you just turn this whole function into async/await instead?

```javascript
for (const fixture of fixtures) {
  await fixture.setup(env);
}
await installRepl(global, env);
```

If not, would consider:

```javascript
let totalPromise = Promise.resolve();
fixtures.forEach((fixture) => {
  totalPromise = totalPromise.then(() => fixture.setup(env));
});
return totalPromise.then(() => {
  installRepl(global, env);
});
```

Reply:
That's a huge improvement. I was originally thinking about maintaining parity with the original `describes.js` file, but I think the win here is worth it.